src/spikeinterface/core/waveform_tools.py

```diff
@@ -397,6 +398,280 @@ def _waveform_extractor_chunk(segment_index, start_frame, end_frame, worker_ctx)
                     wfs[pos, :, :] = wf[:, sparsity_mask[unit_ind]]
 
 
+def extract_waveforms_to_single_buffer(
+    recording,
+    spikes,
+    unit_ids,
+    nbefore,
+    nafter,
+    mode="memmap",
+    return_scaled=False,
+    file_path=None,
+    dtype=None,
+    sparsity_mask=None,
+    copy=False,
+    job_name=None,
```

Comment:
Isn't `job_name` a part of `job_kwargs` ?

Reply:
job_name is for ChunkRecordingExecutor and is not part of job_kwargs.

Reply:
Shouln't it be? 
job_kwargs is basically al the keyword arguments of `ChunkRecordingExecutor`.

Reply:
yes. certainlly. but not in this PR